starfish/core/types/_spot_finding_resutls.py

```diff
@@ -0,0 +1,86 @@
+from typing import List, Mapping, Optional, Tuple
+
+from starfish.core.types import Axes, SpotAttributes
+
+AXES_ORDER = (Axes.ROUND, Axes.CH)
+
+
+class SpotFindingResults:
+    """
+    Wrapper class that describes the results from a spot finding method. The
+    results dict is a collection of (round,ch) indices and their corresponding measured
+    SpotAttributes.
+    """
+
+    def __init__(self, spot_attributes_list: Optional[List[Tuple]] = None):
+        """
+        Construct a SpotFindingResults instance
+
+        Parameters
+        -----------
+        spot_attributes_list : Optional[List[Tuple[indices, SpotAttributes]]]
+            If spots were fond using Imagestack.transform() the result is a list of
+            tuples (indices, SpotAttributes). Instantiating SpotFindingResults with
+            this list will convert the information to a dictionary.
+        """
+        self._results: Mapping[Tuple, SpotAttributes] = dict()
+        if spot_attributes_list:
+            for indices, spots in spot_attributes_list:
+                self._results[indices] = spots
+
+    def set_spots_for_round_ch(self, indices: Mapping[Axes, int], spots: SpotAttributes
+                               ) -> None:
+        """
+        Add the r,ch indices and corresponding SpotAttributes to the results dict.
```

Comment:
```suggestion
        Add the round, ch indices and corresponding SpotAttributes to the results dict.
```

to match what you are doing elsewhere.